### DISCUSSION TITLE
Error Code 409

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#6589] [Questions] [answered]
I received error code 409 from superbase API when I liked and disliked a post.

**Comment by steve-chavez** [comment, marked as answer]
Please provide more details - at minimum the request snippet and the full response error.

**Reply by ghost** [reply]
I got 409 error code when I tried to add one more row in table even though there was a row with same primary key.

So, I got this error.

**Reply by steve-chavez** [reply]
I see. Yes, `409 Conflict` is aptly mapped to a uniqueness violation in PostgreSQL. You can see more details at: https://postgrest.org/en/latest/errors.html#http-status-codes

**Reply by subarnabsadhukhan** [reply]
My issue has been resolved. I appreciate your help. Thank you very much 👍🏻
